Tracker/Scenes/Trackers Flow/TrackerSettings/TrackerSettingsViewController.swift

```diff
@@ -131,10 +150,28 @@ final class TrackerSettingsViewController: UIViewController {
 
     @objc private func textFieldDidChange(_ textField: UITextField) {
         tempTrackerName = textField.text
+        showWarningLabelIfNeeded()
         updateSaveButton()
     }
 
-    func hideKeyboardWhenTappedAround() {
+    private func showWarningLabelIfNeeded() {
+        guard let name = nameTextField.text else { return }
+        if name.count <= Constants.maxNameLength {
+            if !warningLabel.isHidden {
+                settingsTableViewTopConstraint?.constant = 24
+                view.setNeedsLayout()
+            }
+            warningLabel.isHidden = true
+        } else {
+            if warningLabel.isHidden {
```

Comment:
так как внутри нет блока else, то стоит объединить, чтобы лишних операций не делать
```suggestion
        } else if warningLabel.isHidden {
```